frontend/views/pages/DesignSystem.vue

```diff
@@ -1163,6 +1191,11 @@ table {
   }
 }
 
+.c-invite-link-demo {
+  display: flex;
+  align-items: center;
+}
```

Comment:
A component should work visually by itself, without any additional CSS. Adding styles here usually it's a reflection that something is missing to the original component. In this case, the component should have these properties by default. 

Don't forget to review the other places where InvityLink is used and remove duplicated CSS.

Reply:
Fixed!